learn/experimental/vector_search.mdx

```diff
@@ -51,16 +51,21 @@ curl \
       "default": {
         "source":  "openAi",
         "apiKey": "anOpenAiApiKey",
-        "model": "text-embedding-ada-002",
-        "documentTemplate": "A movie titled '{{doc.title}}' whose description starts with {{doc.overview|truncatewords: 20}}"
+        "model": "text-embedding-3-small",
+        "documentTemplate": "A movie titled '{{doc.title}}' whose description starts with {{doc.overview|truncatewords: 20}}",
+        "dimensions": 1536
       }
     }
   }'
 ```
 
 It is mandatory to pass an OpenAI API key through the `OPENAI_API_KEY` environment variable or the `apiKey` field when using an OpenAI embedder. [Generate an API key](https://platform.openai.com/api-keys) from your OpenAI account. Use [tier 2 keys](https://platform.openai.com/docs/guides/rate-limits/usage-tiers?context=tier-two) or above for optimal performance.
 
-`documentTemplate` is an optional field you can use to customize the data you send to the embedder. It is highly recommended you configure a custom template for your documents.
+`model` is a mandatory field indicating a compatible OpenAI model. Meilisearch supports the following [OpenAI models](https://platform.openai.com/docs/models/embeddings): `openai-text-embedding-ada-002`, `text-embedding-3-small`, and `text-embedding-3-large`. 
+
+`documentTemplate` is an optional field. Use it to customize the data you send to the embedder. It is highly recommended you configure a custom template for your documents.
+
+`dimensions` is an optional field. It must be bigger than zero and lower or equal to the chosen model's dimensions. This option is not compatible with `openai-text-embedding-ada-002`. Fewer dimensions may improve performance, but lead to lower search result accuracy.
```

Comment:
> This option is not compatible with `openai-text-embedding-ada-002`.

Technically, if you use it with this model, Meilisearch will do the right thing as long as you pass the exact natural dimensions of this model (1536).

It is OK to simplify like in the text.

Also, typo on model name:
```suggestion
`dimensions` is an optional field. It must be bigger than zero and lower or equal to the chosen model's dimensions. This option is not compatible with `text-embedding-ada-002`. Fewer dimensions may improve performance, but lead to lower search result accuracy.
```

Reply:
Yes, I was aware of the fact `dimensions` would work with `ada-002` if the user were to specify the exact model size. I chose to simplify it to prevent giving users a false choice: "yes, you can specify this setting, but it won't work unless you use the default value."

Thanks for spotting the typo!